perl.h

```diff
@@ -5101,8 +5101,8 @@ Gid_t getegid (void);
         ((what)                                                             \
          ? ((void) 0)                                                       \
          : Perl_croak_nocontext("Assertion %s failed:"                      \
-                                " file \"" __FILE__ "\", line %d",          \
-                                STRINGIFY(what),  __LINE__))
+                                " file \"" __FILE__ "\", line %" LINE_Tf,   \
+                                STRINGIFY(what), (line_t) __LINE__))
```

Comment:
This is fine, though `line_t` is for line numbers stored in COPs (line numbers in perl code) rather than C source.

I doubt it makes a difference unless someone puts `#line 4294967296` (`__LINE__` is now a long) which changes the code from UB to incorrect output.

(doesn't require a change)

Reply:
C99 says `#line`  is UB above 31 bits.  Does that affect this?